azerothcore/azerothcore-wotlk
# refactor(DB/acore_string): Rename locale columns

## CHANGES PROPOSED:
-  Rename content_locN to locale_xxXX for users to easily be able to know which column is for which locale without needing to look in the core. (Hopefully, this will make people want to contribute more to add locales to their language :))



## ISSUES ADDRESSED:



## TESTS PERFORMED:
- Builds
- Worldserver runs



## HOW TO TEST THE CHANGES:

1. apply changes
2. For easy test apply this:
```sql
UPDATE `acore_string` SET `locale_koKR` = '[ACORE] %s', `locale_frFR` = '[ACORE] %s', locale_deDE = '[ACORE] %s', `locale_zhCN` = '[ACORE] %s', `locale_zhTW` = '[ACORE] %s', `locale_esES` = '[ACORE] %s', `locale_esMX` = '[ACORE] %s', `locale_ruRU` = '[ACORE] %s' WHERE `entry` = 3;
```
3. Use a non-english client
4. `.ann This is an announcement to test the new acore_string changes`
5. See that it says [ACORE] your message, instead of [SERVER] your message
## KNOWN ISSUES AND TODO LIST:



## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
